rules/code-quality/src/Naming/MethodCallToVariableNameResolver.php

```diff
@@ -63,14 +63,13 @@ private function getVariableName(MethodCall $methodCall, string $methodCallVarNa
         if ($methodCall->args === [] && $variableName !== null && $variableName !== $methodCallVarName) {
             return $variableName;
         }
-
-        $fallbackVarName = $this->getFallbackVarName($methodCallVarName, $methodCallName);
         $argValue = $methodCall->args[0]->value;
         if ($argValue instanceof ClassConstFetch && $argValue->name instanceof Identifier) {
             return $this->getClassConstFetchVarName($argValue, $methodCallName);
         }
 
         if ($argValue instanceof String_) {
+            $fallbackVarName = $this->getFallbackVarName($methodCallVarName, $methodCallName);
```

Comment:
@TomasVotruba this moving variable is a bug, as it used after `if` at last.

Reply:
Thanks for the info.

The rule should not run at all, as there are many cases to be covered.
I'll try to found out why it's being run